automated install/basic-install.sh

```diff
@@ -319,53 +320,57 @@ It is also possible to use a DHCP reservation, but if you are going to do that,
 setDHCPCD() {
 	# Append these lines to dhcpcd.conf to enable a static IP
 	echo "## interface $piholeInterface
-	static ip_address=$IPv4addr
+	static ip_address=$IPv4_address
 	static routers=$IPv4gw
 	static domain_name_servers=$IPv4gw" | tee -a /etc/dhcpcd.conf >/dev/null
 }
 
 setStaticIPv4() {
+  local IFCFG_FILE
+  local IPADDR
+  local CIDR
 	if [[ -f /etc/dhcpcd.conf ]];then
 		# Debian Family
-		if grep -q "$IPv4addr" /etc/dhcpcd.conf; then
+		if grep -q "$IPv4_address" /etc/dhcpcd.conf; then
 			echo "::: Static IP already configured"
 		else
 			setDHCPCD
-			ip addr replace dev "$piholeInterface" "$IPv4addr"
+			ip addr replace dev "$piholeInterface" "$IPv4_address"
 			echo ":::"
-			echo "::: Setting IP to $IPv4addr.  You may need to restart after the install is complete."
+			echo "::: Setting IP to $IPv4_address.  You may need to restart after the install is complete."
 			echo ":::"
 		fi
 	elif [[ -f /etc/sysconfig/network-scripts/ifcfg-${piholeInterface} ]];then
 		# Fedora Family
 		IFCFG_FILE=/etc/sysconfig/network-scripts/ifcfg-${piholeInterface}
-		if grep -q "$IPv4addr" ${IFCFG_FILE}; then
+		if grep -q "$IPv4_address" "${IFCFG_FILE}"; then
 			echo "::: Static IP already configured"
 		else
-			IPADDR=$(echo ${IPv4addr} | cut -f1 -d/)
-			CIDR=$(echo ${IPv4addr} | cut -f2 -d/)
+			IPADDR=$(echo "${IPv4_address}" | cut -f1 -d/)
+			CIDR=$(echo "${IPv4_address}" | cut -f2 -d/)
 			# Backup existing interface configuration:
-			cp ${IFCFG_FILE} ${IFCFG_FILE}.backup-$(date +%Y-%m-%d-%H%M%S)
+			cp "${IFCFG_FILE}" "${IFCFG_FILE}".backup-"$(date +%Y-%m-%d-%H%M%S)"
 			# Build Interface configuration file:
-			echo "# Configured via Pi-Hole installer" > ${IFCFG_FILE}
-			echo "DEVICE=$piholeInterface" >> ${IFCFG_FILE}
-			echo "BOOTPROTO=none" >> ${IFCFG_FILE}
-			echo "ONBOOT=yes" >> ${IFCFG_FILE}
-			echo "IPADDR=$IPADDR" >> ${IFCFG_FILE}
-			echo "PREFIX=$CIDR" >> ${IFCFG_FILE}
-			echo "GATEWAY=$IPv4gw" >> ${IFCFG_FILE}
-			echo "DNS1=$piholeDNS1" >> ${IFCFG_FILE}
-			echo "DNS2=$piholeDNS2" >> ${IFCFG_FILE}
-			echo "USERCTL=no" >> ${IFCFG_FILE}
-			ip addr replace dev "$piholeInterface" "$IPv4addr"
+			{
+			echo "# Configured via Pi-Hole installer"
+			echo "DEVICE=$piholeInterface"
+			echo "BOOTPROTO=none"
+			echo "ONBOOT=yes"
+			echo "IPADDR=$IPADDR"
+			echo "PREFIX=$CIDR"
+			echo "GATEWAY=$IPv4gw"
+			echo "DNS1=$piholeDNS1"
+			echo "DNS2=$piholeDNS2"
+			echo "USERCTL=no"
+			}>> "${IFCFG_FILE}"
```

Comment:
Yeah, it was either heredoc or grouping and output. Didn't see the benefit of one over the other, actually.